search-sentiments-map/routes/search.js

```diff
@@ -0,0 +1,278 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+/** Server-side script that gets search results from the custom search api. */
+
+const express = require('express');
+let router = express.Router();  // Using Router to divide the app into modules.
+
+const fetch = require('node-fetch'); // Used to access custom search.
+const { Datastore } = require('@google-cloud/datastore');
+const datastore = new Datastore();
+
+const json = require('./../public/country-code.json');
+global.Headers = fetch.Headers;
+
+/** 
+ * Renders a JSON array of the top search results for all countries with API data
+ * obtained every 12 hours for the specified topic.
+ */
+router.get('/:topic', (req, res) => {
+  let topic = req.params.topic;
+  retrieveSearchResultFromDatastore(topic).then(customSearchTopicJsonArray => {
+    // TODO(ntarn): Remove console.log statements when finished debugging.
+    console.log('ntarn debug: In the router.get method'); 
+    console.log(customSearchTopicJsonArray);
```

Comment:
remove console logs

Reply:
I need it for the sentiment chart currently.